src/components/20-molecules/dropdown/index.js

```diff
@@ -354,9 +335,6 @@ class AXADropdown extends NoShadowDOM {
   }
 
   firstUpdated() {
-    // dropdowns cannot meaningfully be initially open, since the underlying
-    // <select> cannot be force-opened natively
-    this.open = false;
     this.dropdown = this.querySelector('.js-dropdown__content');
     this.select = this.querySelector('.js-dropdown__select');
     this.addEventListener('focus', this.onFocus, false);
```

Comment:
i'ts obsolete, because you moved to template?

Reply:
yes, obsolete. Removed.